tests/e2e/es-index-cleaner/es-index.template

```diff
@@ -0,0 +1,5 @@
+      index-prefix: ""
```

Comment:
As far as I understand you use this for adding more things at the end of another `YAML` file, 

Can we use conditions on our templates? I think that would make things more readable. I did something like that in this template:

https://github.com/jaegertracing/jaeger-operator/blob/master/tests/templates/cassandra-jaeger-install.yaml.template#L10